File: SWIFT/Cap04/c04ex05.swift
// c04ex05.swift

import Foundation

print("Entre com um numero equivalente a um mes: ", terminator: "")
if let input = readLine(), let mes = Int(input) {
    switch (mes) {
        case 1:
            print("Janeiro")
        case 2:
            print("Fevereiro")
        case 3:
            print("Março")
        case 4:
            print("Abril")
        case 5:
            print("Maio")
        case 6:
            print("Junho")
        case 7:
            print("Julho")
        case 8:
            print("Agosto")
        case 9:
            print("Setembro")
        case 10:
            print("Outubro")
        case 11:
            print("Novembro")
        case 12:
            print("Dezembro")
        default:
            print("Mes invalido")
    }
} else {
    fatalError("Entrada invalida.")
}

print("\nTecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: KOTLIN/Cap04/c04ex05.kt
// c04ex05.kt

fun main() {
    var Mes: Int = 0

    print("Entre com um numero equivalente a um mes: ")
    Mes = readLine()?.toInt() ?: 0

    when (Mes) {
         1 -> println("Janeiro")
         2 -> println("Fevereiro")
         3 -> println("Marco")
         4 -> println("Abril")
         5 -> println("Maio")
         6 -> println("Junho")
         7 -> println("Julho")
         8 -> println("Agosto")
         9 -> println("Setembro")
        10 -> println("Outubro")
        11 -> println("Novembro")
        12 -> println("Dezembro")
        else -> println("Mes invalido")
    }

    print("\nTecle <Enter> para encerrar... ")
    readLine()
}
